react-stopwatch-component/my-app/src/Stopwatch.tsx

```diff
@@ -0,0 +1,123 @@
+import { FaPlay, FaPause } from 'react-icons/fa';
+import './Stopwatch.css';
+import { useState } from 'react';
+
+export default function Stopwatch() {
+  const [time, setTime] = useState(0);
+  const [running, setRunning] = useState(false);
```

Comment:
You could calculate `running` from `intervalId` if you wanted. Then you wouldn't need to worry about keeping them in sync.